search_server.py

```diff
@@ -29,7 +29,8 @@
 _STYLE_SKIP = ""
 _CLOSE_STYLE_GOOD = "[/]" if _STYLE_GOOD else ""
 _CLOSE_STYLE_SKIP = "[/]" if _STYLE_SKIP else ""
-
+_DEFAULT_URL_REQUEST_TIMEOUT = 15 # seconds
+_TRUNCATE_TEXT_BYTES = 2048
```

Comment:
How did you get to the number 2048?

Reply:
I would rather you make `_TRUNCATE_TEXT_BYTES` a command-line parameter for `serve`, with a default value. Whether that value should be `None` or some number .. I'm not convinced yet.

Reply:
I guess `_DEFAULT_URL_REQUEST_TIMEOUT` should maybe also be a parameter with a default value, but that is less important